### DISCUSSION TITLE
use custom search button

### DISCUSSION THREAD

**Discussion by oben01** [KevinVandy/mantine-react-table#12] [Q&A] [answered]
Hello folks is it possible to use a custom search button ? 
![2023-02-25_18-23](https://user-images.githubusercontent.com/22369027/221370845-880f22f4-d191-41ce-a9cc-ad77ba0a293e.png)

**Comment by KevinVandy** [comment, marked as answer]
You got a couple options. If you just want to change out the icons, see the [Custom Icons Guide](https://www.mantine-react-table.com/docs/guides/custom-icons). Or if you want to control how all of the icon buttons are rendered, see the [Toolbar Customization guide in Internal Actions section](https://www.mantine-react-table.com/docs/guides/toolbar-customization#customize-built-in-internal-toolbar-button-area)

**Reply by oben01** [reply]
Thank u :+1:
